subscription/faq.md

```diff
@@ -1,381 +1,45 @@
 ---
-description: Subscription & Billing FAQs
-keywords: Docker, Docker Hub, billing FAQs, pull rate limiting FAQs, subscription, platform
-title: Subscription & Billing FAQs
-toc_min: 1
-toc_max: 2
-redirect_from:
-- /docker-hub/billing/faq/
+description: Subscription FAQs
+keywords: Docker, Docker Hub, subscription FAQs, subscription, platform
+title: Subscription FAQs
 ---
 
-## Subscription tiers
+### I want to buy a large number of Docker Business subscription seats. How do I do this? Can I get a discount?
 
-### What subscriptions levels are available for Docker?
+Docker offers volume pricing for Docker Business subscriptions starting at 100 seats. Contact the [Docker Sales Team](https://goto.docker.com/pricing-question.html){:target="blank" rel="noopener" class=""} for more information.
```

Comment:
This link seems dead, perhaps this is a better link? https://www.docker.com/company/contact/

Reply:
According to https://www.notion.so/dockerinc/Paid-Products-f024230ed0424b358ac0c18da9f234c5, it should be `starting at 25 seats` and the direct link to the contact form is https://www.docker.com/pricing/contact-sales/